src/macaron/repo_finder/commit_finder.py

```diff
@@ -211,17 +211,12 @@ def extract_commit_from_version(git_obj: Git, version: str) -> tuple[str, str]:
             logger.debug("Failed to retrieve commit: %s", error)
 
     if not commit:
-        return "", ""
-
-    branch_name = _get_branch_of_commit(commit)
-    if not branch_name:
-        logger.debug("No valid branch found for commit: %s", commit.hash)
-        return "", ""
+        return ""
 
-    return branch_name, commit.hash
+    return str(commit.hash)
```

Comment:
What if `commit.hash` is `None`? Should we check for that?

Reply:
I don't believe it can be `None`. The `init` method in the class used by `GitPython` for "Blobs, Trees, Commits and Tags" has an `assert` statement that checks the `binsha` is 20 characters in length.

Reply:
In that case, I don't think creating a string object from `commit.hash` is necessary.

You can rewrite this line and the if statement above like this:
```suggestion
    return commit.hash if commit else None
```